apps/spruce/src/constants/routes.ts

```diff
@@ -48,6 +48,12 @@ export enum ProjectSettingsTabRoutes {
   GithubPermissionGroups = "github-permission-groups",
 }
 
+export enum AdminSettingsTabRoutes {
+  General = "general",
+  RestartTasks = "restart-tasks",
+  EventLogs = "event-log",
```

Comment:
```suggestion
  EventLogs = "event-logs",
```

Reply:
you can keep it as `event-log`, but the name should be changed from `EventLogs` => `EventLog`